force-app/main/default/classes/ContactController.cls

```diff
@@ -1,170 +1,132 @@
 public with sharing class ContactController {
```

Comment:
please go through whole class regarding naming for variables, it is not good name like "c" (even as loop variable) or "source", try to make them meaningful like "contacts", "accountToContacts", "inputContact" etc...

Reply:
Fixed